CHANGELOG.md

```diff
@@ -1,5 +1,148 @@
+
+  - Parameters can be added from YAML file in reforms. The function `legislations.load_file()` loads a YAML file. The function `Node.add_child()` adds a new child to an existing legislation node.
+
+    Example:
+    ```python
+    import os
+    from openfisca_core import legislations
+
+    dir_path = os.path.dirname(__file__)
+
+    def reform_modify_legislation_(legislation):
+        file_path = os.path.join(dir_path, 'plf2016.yaml')
+        reform_legislation_subtree = legislations.load_file(name='plf2016', file_path=file_path)
+        legislation.add_child('plf2016', reform_legislation_subtree)
+        return legislation
+
+    ...
+    ```
+  
+  - Although loading a YAML file is now the prefered way to extend a legislation for a reform, new nodes can still still can be defined dynamically. However the syntax is different. The constructors of the classes `Node`, `Parameter` and `ValueAtInstant` must be used instead of providing a raw python object.
+
+    Before :
+    ```python
+    reform_legislation_subtree = {
+        "@type": "Node",
+        "description": "PLF 2016 sur revenus 2015",
+        "children": {
+            "decote_seuil_celib": {
+                "@type": "Parameter",
+                "description": "Seuil de la décôte pour un célibataire",
+                "format": "integer",
+                "unit": "currency",
+                "values": [
+                    {'start': u'2016-01-01', },
+                    {'start': u'2015-01-01', 'value': round(1135 * (1 + inflation))},
+                    ],
+                },
+            "decote_seuil_couple": {
+                "@type": "Parameter",
+                "description": "Seuil de la décôte pour un couple",
+                "format": "integer",
+                "unit": "currency",
+                "values": [
+                    {'start': u'2065-01-01', },
+                    {'start': u'2015-01-01', 'value': round(1870 * (1 + inflation))},
+                    ],
+                },
+            },
+        }
+    reference_legislation_copy['children']['plf2016_conterfactual'] = reform_legislation_subtree
+    ```
+
+    Now:
+    ```python
+    from openfisca_core.legislations import Node, Parameter, ValueAtInstant
+
+    reform_legislation_subtree = Node('plf2016_conterfactual', children = {
+        'decote_seuil_celib': Parameter('decote_seuil_celib', values_list = [
+            ValueAtInstant('decote_seuil_celib', "2015-01-01", value=round(1135 * (1 + inflation))),
+            ValueAtInstant('decote_seuil_celib', "2016-01-01", value=None),
+            ]),
+        'decote_seuil_couple': Parameter('decote_seuil_couple', values_list = [
+            ValueAtInstant('decote_seuil_couple', "2015-01-01", value=round(1870 * (1 + inflation))),
+            ValueAtInstant('decote_seuil_couple', "2065-01-01", value=None),
+            ]),
+        })
+    reference_legislation_copy.add_child('plf2016_conterfactual', reform_legislation_subtree)
+    ```
```

Comment:
This example demonstrates the creation of parameters computed dynamically. Showing a computation is important here.